### ISSUE TITLE
mae and RMSE

### ISSUE THREAD

**Issue by dontlearncpp** [niki-amini-naieni/CounTX#4]
Nice work!
Thanks for sharing the code！
with your train command:
the result are 
val:Test MAE: 21.80, Test RMSE: 73.82 
test:Test MAE: 20.23, Test RMSE: 106.87 
with cuda 11.8 torch 2.0.0 GPU 4090
In you code, the MAE and RMSE for the validation set was Calculated for each epoch. but, I change it into every 20 epochs
and the batch size was set to 48, 1000 epochs.
For reproducing your accuracy, should I change the batch size into 8, and Calculate MAE and RMSE for each epoch?

**Comment by niki-amini-naieni** [comment]
Hi, yes, you should change the batch size to 8 and calculate the MAE and RMSE for each epoch (for early stopping purposes). Please let me know if you have any further questions.
